deployment/ansible/roles/driver.access/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Set table-level access privileges for Windshaft DB role
+  postgresql_privs: >
+    state=present
+    type=table
+    priv=SELECT
+    objs="{{ windshaft_tables }}"
```

Comment:
This attribute asks for a comma delimited list, which can be provided with `{{ windshaft_tables | join(",") }}`.